app/models/site.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class Site
+  def self.banner_message
+    @banner_message ||= Settings.site_banner_message
```

Comment:
To prevent issues with blank values like the empty string maybe we could do here:
`@banner_message ||= Settings.site_banner_message.presence`

Reply:
Done!